### DISCUSSION TITLE
A LITTLE QUESTION

### DISCUSSION THREAD

**Discussion by guiworldtv2** [streamlink/streamlink#4649] [Q&A] [answered]
HEY FRIENDS, I WOULD LIKE A HELP IF POSSIBLE, I'M INSTALLING STREAMLINK ON MY LINUX, I DON'T WANT IT TO OPEN THE TRANSMISSIONS IN VLC BUT GENERATE A TRANSMISSION LINK TO A .M3U FILE, DOES ANYONE KNOW IF IT IS POSSIBLE TO SAY THAT? I WANT THE M3U FILE TO WATCH ON KODI ON ANOTHER DEVICE, THANKS

**Comment by bastimeyer** [comment, marked as answer]
https://streamlink.github.io/cli.html#cmdoption-stream-url
